### PR TITLE
Remove references to master table in favour of Data Engineering convention

### PR DESCRIPTION
**Pull request by datajoely**
Updated spaceflights tutorial to use language in line with our Data Engineering convention.

### CODE REVIEW

**Review by yetudada — APPROVED** [review]
👏  Thank you @datajoely!

File: spaceflights/{{ cookiecutter.repo_name }}/src/{{ cookiecutter.python_package }}/pipelines/data_processing/nodes.py [hunk 1]
```diff
@@ -73,20 +73,24 @@ def preprocess_shuttles(shuttles: pd.DataFrame) -> pd.DataFrame:
...
-    """Combines all data to create a master table.
+    """Combines all data to create a model input table.
 
     Args:
         shuttles: Preprocessed data for shuttles.
         companies: Preprocessed data for companies.
         reviews: Raw data for reviews.
     Returns:
-        Master table.
+        Mode input table.
```
Comment (MerelTheisenQB) [inline spaceflights/{{ cookiecutter.repo_name }}/src/{{ cookiecutter.python_package }}/pipelines/data_processing/nodes.py:86]:
```suggestion
        Model input table.
```

**Review by MerelTheisenQB — APPROVED** [review]
👍

File: spaceflights/{{ cookiecutter.repo_name }}/src/{{ cookiecutter.python_package }}/pipelines/data_processing/nodes.py (hunk 1 above)
Comment (datajoely) [reply spaceflights/{{ cookiecutter.repo_name }}/src/{{ cookiecutter.python_package }}/pipelines/data_processing/nodes.py]: Thank you - good catch!

**Review by stichbury — APPROVED** [review]

**Review by ignacioparicio — APPROVED** [review]

**Pull request merged**